change/@ni-nimble-tokens-3974f217-50f5-4e6f-a318-be960d230b62.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Default Buttons Updated Styling",
```

It's often good to have different comments for each package so that clients who use just that package can see more specifically what changed.
```suggestion
  "comment": "Changed button label font weight token from Regular to Semi-Bold",
```